ui/index.d.ts

```diff
@@ -631,7 +631,17 @@ export interface UIExtensionVariables {
     reporter: TelemetryReporter | undefined;
 }
 
+export interface IAddUserAgent {
+    addUserAgentInfo(additionalUserAgentInfo: any): void;
+}
+
+/**
+ * Retrieves a user agent string specific to the VS Code extension, of the form `${extensionName}/${extensionVersion}`,
+ * and appends it to the given user agent string, if it isn't already in the string.
+ */
+export declare function appendExtensionUserAgent(userAgent: string | undefined): string;
+
 /**
- * Adds a user agent specific to the VS Code extension, of the form `${extensionName}/${extensionVersion}`
+ * Adds the extension user agent to the given ServiceClient or other object support AddUserAgentInfo
  */
-export declare function addExtensionUserAgent(client: ServiceClient): void;
+export declare function addExtensionUserAgent(client: IAddUserAgent): void;
```

Comment:
The function definition doesn't match with extensionUserAgent.ts anymore.  Will that matter?

Reply:
Fixed, thanks.